### PR TITLE
Add license check

### PR DESCRIPTION
**Pull request by heyanlong**
#### What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### What does this PR do?
Add license check

#### Which issue(s)/PR(s) this PR relates to?
None

### CODE REVIEW

**Review by yixinglu — APPROVED** [review]
LGTM

**Comment by Shylock-Hg** [comment]
Run in CI?

**Comment by heyanlong** [comment]
Request CI

**Review by Shylock-Hg — APPROVED** [review]

**Comment by heyanlong** [comment]
Ignore the license verification of cmake, conf, tests, and docker for now, and you can open it if necessary

**Comment by heyanlong** [comment]
Need fix build error for self-hosted

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -65,6 +67,8 @@ jobs:
         with:
           run: sh -c "find . -mindepth 1 -delete"
       - uses: actions/checkout@v2
+      - name: Check License Header
+        uses: apache/skywalking-eyes@main
```
Comment (yixinglu) [inline .github/workflows/pull_request.yml:71]: This job no longer needs to check the license repeatedly, because the `lint` job has already checked it.

**Review by yixinglu — APPROVED** [review]

**Review by Aiee — APPROVED** [review]

**Pull request merged**